services_app/src/main/java/org/opendatakit/services/preferences/activities/AppPropertiesActivity.java

```diff
@@ -101,12 +110,16 @@ protected void onCreate(Bundle savedInstanceState) {
     preferenceViewModel.getAdminMode().setValue(mAdminMode);
 
     preferenceViewModel.getAdminMode().observe(this, new Observer<Boolean>() {
+      @SuppressLint("WrongConstant")
       @Override
       public void onChanged(Boolean adminMode) {
         int titleResId = adminMode ?
                 R.string.action_bar_general_settings_admin_mode :
                 R.string.action_bar_general_settings;
 
+
```

Comment:
In general commented out code should not be checked in. If there is a reason to leave commented code in please put in a comment what the code is useful for that is commented out.

Reply:
Yes I commented the code out because It wasn't useful at that moment. It's a little bit buggy. I'll clean it up in the next PR